circus/process.py

```diff
@@ -11,7 +11,8 @@
 import sys
 import errno
 import os
-from subprocess import PIPE
+from subprocess import PIPE, CREATE_NEW_PROCESS_GROUP
```

Comment:
`CREATE_NEW_PROCESS_GROUP` isn't defined on non-linux hosts, so this should be in a try/except.